src/file_organizer/config/provider_env.py

```
@@ -315,7 +315,7 @@ def _get_model_configs_from_profile(
             vision_cfg.name,
         )
         return text_cfg, vision_cfg
-    except Exception:
+    except Exception:  # Intentional catch-all: config loading involves YAML parsing, imports, etc.
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)